### ISSUE TITLE
ISSN 1783-1814 | Studies in Spirituality (Peeters) | keine Einspielungen

### ISSUE THREAD

**Issue by IxKo** [ubtue/DatenProbleme#2069]
#### URL
https://poj.peeters-leuven.be/RSS/SIS.xml
    
### Problembeschreibung

Der RSS-Feed hat sich am 02.02.24 aktualisiert.
Auf ub28 wird die Zs. aber weiterhin als Status rot (Error) dargestellt und bei TRY RSS wird eine gelbe leere Seite als Ergebnis ausgespuckt.

**Comment by jriedl** [comment]
Das liegt an der fehlschlagenden Konvertierung des gelieferten `date`-Feldes. Hier wird `2022-2023` zurückgegeben. Mit `zotero_strptime_format    = %Y|%Y-%Y`  (in Additional Config zum Testen oder direkt im Abschnitt für diese Zeitschrift in der conf) werden Records erzeugt, als Jahr ergibt sich dann 2023 (264$c, 773$g, 936$j). Ist das durch die Erfassungsrichtlinien gedeckt? Einführen einer Sonderbehandlung im Code und Mitführen der Bereiche würde voraussichtlich insbesondere im Zusammenhang mit Sortierfeldern Probleme machen.

**Comment by IxKo** [comment]
> Ist das durch die Erfassungsrichtlinien gedeckt?
Nein, als Erscheinungsjahr sollte nur eine Jahresangabe angegeben werden. In diesem Fall 2023.

Mit dem halbautomatischen Zotero-Translator ubtue_Peeters wird das Datum umgewandelt zu 2022, was auch nicht korrekt wäre.

Korrekt wäre:
264$c 2023 (Pica-Feld 1100)
773$g 2022/2023 
936$j 2022/2023 (Pica-Feld 4070 $j)

**Comment by jriedl** [comment]
> Nein, als Erscheinungsjahr sollte nur eine Jahresangabe angegeben werden. In diesem Fall 2023

Aber genau das ist doch der Fall.

Dass 936$j Doppelbelegungen zulässt, überrascht mich sehr, da wir das seit langem als Sortierfeld verwenden, in 264$c finde ich dagegen häufig solche Doppeleinträge im IxTheo-Bestand (z.B. 596316151, .1762722585) Anderesherum gefragt: Ist der Fall in den vergangenen Jahren so häufig aufgetreten, dass die Integration einer Spezialbehandlung hier sinnvoll erscheint oder beschränkt sich das auf so wenige Fälle, dass es vielleicht hinreichend ist, die mit 2023 einspielen zu lassen und an den entsprechenden Stellen manuell zu korrigieren?

Mit der Korrektur des `zotero_strptime_format    = %Y|%Y-%Y` in der conf sollte dann auch nichts mehr rot sein.

**Comment by IxKo** [comment]
Meines Wissens darf als **Erscheinungsdatum** nur eine Jahresangabe stehen. Nur bei fortlaufenden Ressourcen oder mehrteiligen Monografien stehen zwei Angaben, aber getrennt mit einem Unterfeld, da es Anfang und Ende angibt, da es ein Verlauf ist.
Ein Aufsatz erscheint nur in einem Jahr und nicht über zwei oder mehr Jahre.
Das Feld 936$j gibt hingegen das **Berichtsjahr** an. Dies bezieht sich auf die Quelle, aus welcher das unselbstständige Werk entstammt (=Band/Heft) und hier kann eine doppelte Jahresangabe vorkommen.


Mir ging es in erster Linie darum, dass der rote Status nicht stimmen kann, da ein neues Heft erschienen ist.
> Mit der Korrektur des zotero_strptime_format = %Y|%Y-%Y in der conf sollte dann auch nichts mehr rot sein.
Falls wir diesen Weg gehen - soll ich die Anpassung machen oder möchten Sie das selbst tun?


Danke, ich habe mit TRY RSS und conf-Änderung "zotero_strptime_format    = %Y|%Y-%Y" getestet und es scheint zu klappen.
Daher habe ich es direkt selbst in der conf nachgetragen und prüfe morgen die Einspielung.
